experimental/include/experimental/Transforms/LSQSizing/HandshakeSizeLSQs.h

```diff
@@ -0,0 +1,30 @@
+//===- HandshakeSizeLSQs.h - Sizes the LSQs --------*- C++ -*-===//
```

Comment:
Insert our standard copyright notice and make the first line 80-characters long. Please make sure this is the case in all new files you added.